config/thermofloor/heatit20a.xml

```diff
@@ -0,0 +1,225 @@
+      <Item label="3 LED flashing at ½ second interval (1 Hz)." value="3"/>
+    </Value>
+    <Value genre="config" index="2" instance="1" label="Specifies the brightness level of the LED when it is on.(0-100) Default is 50." size="1" type="byte" value="50">
+      <Help>Set Brightnes</Help>
+    </Value>
+    <Value genre="config" index="3" instance="1" label="Thermistor type connected to input 1." size="1" type="list" value="0">
+      <Help>This parameter decides which kind of thermistor that is connected to the input.</Help>
+      <Item label="0 No thermistor, input is disabled. " value="0"/>
+      <Item label="1 10K NTC. (PART NUMBER: TT02-10KC3-93D-3000R-TPH). (Default)" value="1"/>
+    </Value>
+    <Value genre="config" index="4" instance="1" label="Thermistor type connected to input 2." size="1" type="list" value="0">
+      <Help>This parameter decides which kind of thermistor that is connected to the input.</Help>
+      <Item label="0 No thermistor, input is disabled. " value="0"/>
+      <Item label="1 10K NTC. (PART NUMBER: TT02-10KC3-93D-3000R-TPH). (Default)" value="1"/>
+    </Value>
+    <Value genre="config" index="5" instance="1" label="Thermistor type connected to input 3." size="1" type="list" value="0">
+      <Help>This parameter decides which kind of thermistor that is connected to the input.</Help>
+      <Item label="0 No thermistor, input is disabled. " value="0"/>
+      <Item label="1 10K NTC. (PART NUMBER: TT02-10KC3-93D-3000R-TPH). (Default)" value="1"/>
+    </Value>
+    <Value genre="config" index="6" instance="1" label="Thermistor type connected to input 4." size="1" type="list" value="0">
+      <Help>This parameter decides which kind of thermistor that is connected to the input.</Help>
+      <Item label="0 No thermistor, input is disabled. " value="0"/>
+      <Item label="1 10K NTC. (PART NUMBER: TT02-10KC3-93D-3000R-TPH). (Default)" value="1"/>
+    </Value>
+    <Value genre="config" index="7" instance="1" label=" Input 1 calibration." size="1" type="byte"  max="255" min="0" size="1" type="byte" value="0">
+      <Help>-40 - 40 (-4.0C - 4.0C) Default is 0 (0.0C) To set a negative value, use 255 and subtract the desired value.</Help>
+    </Value>
+    <Value genre="config" index="8" instance="1" label=" Input 2 calibration." size="1" type="byte"  max="255" min="0" size="1" type="byte" value="0">
+      <Help>-40 - 40 (-4.0C - 4.0C) Default is 0 (0.0C) To set a negative value, use 255 and subtract the desired value.</Help>
+    </Value>
+    <Value genre="config" index="9" instance="1" label=" Input 3 calibration." size="1" type="byte"  max="255" min="0" size="1" type="byte" value="0">
+      <Help>-40 - 40 (-4.0C - 4.0C) Default is 0 (0.0C) To set a negative value, use 255 and subtract the desired value.</Help>
+    </Value>
+    <Value genre="config" index="10" instance="1" label=" Input 4 calibration." size="1" type="byte"  max="255" min="0" size="1" type="byte" value="0">
```

Comment:
This should not be required. It should be auto discovered.

Reply:
Do you mean only this particular line, or the entire "  <CommandClass id="112">" ?